app/dev/index.html

```diff
@@ -0,0 +1,248 @@
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+
```

Comment:
What's this?

Reply:
recreation of index.php from prototype
so far i did halfway only, but merge in first so others can continue when they want

at least could faster finish

Reply:
divs spammed to test scrolling bc of sticky navbar, will be removed when more content filled in. probably tonight

Reply:
Ok, I'm also working on the main page, probably will do a pr once I got the structure

Reply:
which part you working?

Reply:
**I'm working on:** 
- home & signin & signup pages (covering both HTML and CSS)

**I just finished:**
- layout files such as header and footer.
- a global CSS file that will be included in all pages.

If you guys have already did or going to do some of these works, just go ahead, cause I will be finishing these on today.